src/ol_infrastructure/components/services/vault.py

```diff
@@ -300,30 +300,80 @@ def __init__(
 
         resource_options = ResourceOptions(parent=self).merge(opts)  # type: ignore
 
+        # Create the pki-intermediate-ca endpoint / mount
         self.pki_intermediate_ca_backend = Mount(
             "pki-intermediate-ca",
             opts=resource_options,
             path="pki-intermediate-ca",
             type="pki",
-            description="Backend to create certs for pki-intermediate-env backends",
+            description="Backend to create certs for pki-intermediate-{env} backends",
```

Comment:
Missing the `f` prefix if this is intended to be a formatted string

Reply:
It is not intended to be a f-string.